.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: App CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
```

Comment:
Could we add codecov.io and Slack posting support to this? I'd like to maintain parity with what we had in Travis.

Reply:
Let's skip adding codecov and Slack for now. Just remove the `yarn run format` step for now and we can add those in a follow-up. Go ahead and remove the `travis.yml` file since we don't need both anymore and we can merge this.

Reply:
updated now in [this commit](https://github.com/internetarchive/iaux-typescript-wc-template/pull/2/commits/b9fad04b0d87befd657528ac5cb2035c3db986a9)